circle.yml

```diff
@@ -136,10 +136,6 @@ stages:
           cp $DEVOPS_WORKDIR/environments/$DEPLOY_ENV ./.env
           cp $DEVOPS_WORKDIR/keys/.googlecloudkey.$DEPLOY_ENV.json ./.googlecloudkey.json
 
-      - type: shell
```

Comment:
I thought about this some. The reason this test stage existed was to make sure we couldn't check in code that wouldn't build in production. It's been some time since we've seen a failure like this (thank goodness!) but I think it's a good invariant to want to preserve.

What if we only ran this stage when `"${CIRCLE_BRANCH}" == "master"`?

Reply:
With the deploy flag, we essentially have 3 configs: dev, prod, and deploy.
- We test the dev config during PR review.
- We test the deploy config during CI to dev, staging, and prod
- A passing prod config can still fail deploy.
- By removing SSR, we no longer have a node config (which was where the errors came from)